resources/sql/queries.sql

```diff
@@ -670,6 +670,12 @@ VALUES (:user, :userattrs::jsonb)
 ON CONFLICT (userId)
 DO UPDATE SET userAttrs = :userattrs::jsonb;
 
+-- :name update-user-settings! :!
+INSERT INTO user_settings (userId, settings)
+VALUES (:user, :settings::jsonb)
+ON CONFLICT (userId)
+DO UPDATE SET settings = :settings::jsonb;
```

Comment:
If you change this to "DO NOTHING", none of the tests will detect it.

Reply:
The test was there previously (due to your previous review comment) but got lost in the latest patch. Done.